build.sbt

```diff
@@ -44,6 +44,13 @@ lazy val core = (project in file("core"))
     testOptions in Test ++= Seq(Tests.Argument(TestFrameworks.ScalaTest, "-o")))
   .configs(MultiJvm)
 
+lazy val reconciler = (project in file("reconciler"))
+  .enablePlugins(Common, AutomateHeaderPlugin, SbtOsgi, MultiJvmPlugin)
+  .dependsOn(core % "test->test;compile->compile", session)
+  .settings(
+    libraryDependencies ++= Dependencies.reconcilerDependencies
```

Comment:
set `name` and testOptions as in above `core` project? also Automatic-Module-Name